openproblems/tasks/_cell_cell_communication/cell_cell_communication_ligand_target/metrics/auprc.py

```diff
@@ -0,0 +1,8 @@
+from .....tools.decorators import metric
+from ..._common.metrics import auprc as _auprc
+from ..api import MERGE_KEYS
+
+
+@metric(**_auprc.metadata)
+def auprc(adata):
```

Comment:
Why not just use `adata.uns['merge_keys']`?

Reply:
Hmmm, but then do we actually need the definitions of `MERGE_KEYS`? Do we get rid of those all together?
(MERGE_KEYS are used to assert adata.uns['merge_keys'] is correct)

Or do you mean only for the case of the metrics?

Reply:
Yes, I mean just for the metrics. MERGE_KEYS is required to define the API but if you draw straight from uns in the metric then you don't need to define the metric separately for each subtask